nucypher/policy/conditions/lingo.py

```diff
@@ -212,6 +224,121 @@ def __init__(self, operand: AccessControlCondition):
 }
 
 
+# CONDITION_VARIABLE = {
+#     "varName": STR,
+#     "condition": {
+#         CONDITION
+#     }
+# }
+#
+# SEQUENTIAL_CONDITION = {
+#     "name": ...  (Optional)
+#     "conditionType": "sequential",
+#     "conditionVariables": [CONDITION_VARIABLE*]
+# }
+class ConditionVariable(_Serializable):
+    class Schema(CamelCaseSchema):
+        var_name = fields.Str(required=True)  # TODO: should this be required?
+        condition = _ConditionField(required=True)
+
+        @post_load
+        def make(self, data, **kwargs):
+            return ConditionVariable(**data)
+
+    def __init__(self, var_name: str, condition: AccessControlCondition):
+        self.var_name = var_name
+        self.condition = condition
+
+
+class SequentialAccessControlCondition(MultiConditionAccessControl):
+    CONDITION_TYPE = ConditionType.SEQUENTIAL.value
+    MAX_NUM_CONDITIONS = 5
+    MAX_MULTI_CONDITION_NESTED_LEVEL = 2
```

Comment:
We can re-parametrize these limits as we encounter more use cases involving sequential calls/conditions, and weigh the implied fee revenue against risk of misuse